### DISCUSSION TITLE
Opposite of 'Ignored filters values'

### DISCUSSION THREAD

**Discussion by Exotelis** [spatie/laravel-query-builder#493] [Ideas]
Hi,

I'd like filter for a user role. It would be great if an exception would be thrown if an invalid filter value has been submitted.

Let's say a user can have one of the following roles `Admin, Mod, User`. 

When I submit `GET /users?filter[role]=Guest`  an exception should be thrown with the message that Guest is invalid. This isn't possible as of yet, is it?

**Comment by AlexVanderbist** [comment]
I'm afraid not. However this might be a bit out of scope for this package. You can add such a "guard" to your controller by checking the request parameters for this filter before calling any `QueryBuilder` methods.
